components/common/CustomInput/CustomInput.tsx

```diff
@@ -0,0 +1,59 @@
+"use client";
+
+import { useRef } from "react";
+import useInputValidation from "hooks/useInputValidation";
+import { ValidationTarget } from "types/enums/inputValidation.enum";
+import UserInput from "./UserInput";
+import styles from "./CustomInput.module.scss";
+
+interface ValidationType {
```

Comment:
```suggestion
interface IValidationType {
```

interface는 prefix로 대문자 I 붙여주세요~!

Reply:
넵!